maven-plugin/src/main/java/net/revelc/code/formatter/FormatterMojo.java

```diff
@@ -583,7 +604,7 @@ private void createCodeFormatter() throws MojoExecutionException {
 		try {
 			this.resourceManager.addSearchPath(FileResourceLoader.ID,
 							this.basedir.getAbsolutePath());
-			configInput = this.resourceManager.getResourceAsInputStream(newConfigFile);
+			configInput = this.resourceManager.getResourceAsInputStream(this.configFile);
```

Comment:
This was the bug I was mentioning.  The 'this' reference here is to the wrong field which would otherwise break usage for javascript anyway.  The this should be removed.  It should be using the 'configFile directly passed into the private method.